### DISCUSSION TITLE
Planned release date for v.1.9.0

### DISCUSSION THREAD

**Discussion by gerold-csendes-epam** [TDAmeritrade/stumpy#430] [Help: Other Questions] [answered]
Hi stumpy folks,
is there a planned release date for v.1.9.0? I tried searching for this info, sorry if it is available somewhere, I just couldn't find it. Thanks a lot!

**Comment by seanlaw** [comment, marked as answer]
@gerold-csendes-epam I think the short answer is "very soon". There's one particular PR that we're trying to squeeze into v1.9.0 but it will require a bit of work. If you are interested, you can install the latest development version from source and have access to the bleeding-edge features. It is currently stable and fully unit tested.

**Reply by gerold-csendes-epam** [reply]
fantastic, thanks @seanlaw
